apps/wallet/src/ui/app/components/ledger/ConnectLedgerModal.tsx

```diff
@@ -42,22 +53,50 @@ export function ConnectLedgerModal({ onClose, onConfirm, onError }: ConnectLedge
             <DialogContent containerId="overlay-portal-container">
                 <Header title="Connect Ledger Wallet" onClose={onClose} titleCentered />
                 <DialogBody>
-                    <div className="flex flex-col items-center gap-y-lg">
-                        <div className="p-md">
+                    <div className="flex flex-col items-center gap-y-md">
+                        <div className="flex w-full justify-center p-md">
                             <LedgerLogo />
                         </div>
-                        <span className="text-center text-body-lg text-neutral-40 dark:text-neutral-60">
+                        <span className="text-center text-body-md text-neutral-40 dark:text-neutral-60">
                             Connect your ledger to your computer, unlock it, and launch the IOTA
                             app. Click Continue when done.
                         </span>
+                        <InfoBox
+                            title="Update your IOTA Ledger app"
```

Comment:
Maybe it should be something like "Make sure you have the latest...."

Because right not it feels like the ledger is always outdated